<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rules</class>
 <widget class="QDialog" name="rules">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>480</y>
     <width>150</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: #FFFF33;background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #6CC9EB, stop: 1 #0d5ca6); 
font: 26pt &quot;HanziPen SC&quot;;
border-width: 5px;
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>&lt; previous</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>171</width>
     <height>131</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>300</width>
     <height>300</height>
    </size>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>70</y>
     <width>171</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_2">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>480</y>
     <width>150</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: #FFFF33;background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #6CC9EB, stop: 1 #0d5ca6); 
font: 26pt &quot;HanziPen SC&quot;;
border-width: 5px;
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>next &gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>390</y>
     <width>340</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HanziPen SC</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;HanziPen SC&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>390</y>
     <width>340</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HanziPen SC</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;HanziPen SC&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>530</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: #FFFF33;background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #6CC9EB, stop: 1 #0d5ca6); 
font: 26pt &quot;HanziPen SC&quot;;
border-width: 5px;
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>Back to main</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
